src/Raven.Studio/typescript/components/common/Buttons.stories.tsx

```diff
@@ -0,0 +1,178 @@
+                            <Button color="node" size="xs">
+                                node
+                            </Button>
+                            <Button color="shard" size="xs">
+                                shard
+                            </Button>
+                        </Col>
+                    </Row>
+                    <Row className="mt-3">
+                        xs size, outline
+                        <Col className="d-flex gap-1">
+                            <Button color="primary" size="xs" outline>
+                                primary
+                            </Button>
+                            <Button color="secondary" size="xs" outline>
+                                secondary
+                            </Button>
+                            <Button color="danger" size="xs" outline>
+                                danger
+                            </Button>
+                            <Button color="dark" size="xs" outline>
+                                dark
+                            </Button>
+                            <Button color="info" size="xs" outline>
+                                info
+                            </Button>
+                            <Button color="light" size="xs" outline>
+                                light
+                            </Button>
+                            <Button color="progress" size="xs" outline>
+                                progress
+                            </Button>
+                            <Button color="success" size="xs" outline>
+                                success
+                            </Button>
+                            <Button color="warning" size="xs" outline>
+                                warning
+                            </Button>
+                            <Button color="node" size="xs" outline>
+                                node
+                            </Button>
+                            <Button color="shard" size="xs" outline>
+                                shard
+                            </Button>
+                        </Col>
+                    </Row>
+                </>
+            )}
+            {args.buttonGroup && (
+                <>
+                    <div>
+                        <ButtonGroupWithLabel label="Button group with label">
+                            <Button color="danger">
+                                <i className="icon-trash"></i>
+                                <span>Delete</span>
+                            </Button>
+                            <Button>
+                                <i className="icon-lock"></i>
+                                <span>Set delete lock mode</span>
+                            </Button>
+                        </ButtonGroupWithLabel>
+                    </div>
+                </>
+            )}
+        </>
+    );
+};
+
+export const GroupWithLabel = boundCopy(Template);
```

Comment:
If you have 2 different stories to present then no need to use bound copy of template. Just create 2 different stories and export them. Please check other files for reference.

Reply:
I've split the story into two separate.